app/core/store/modules/me.coffee

```diff
@@ -32,6 +32,9 @@ module.exports = {
     isSchoolAdmin: (state) ->
       (state?.permissions or []).indexOf('schoolAdministrator') > -1
 
+    preferredLocale: (state) ->
```

Comment:
This is amazing. Great example of using vuex instead of the global `me` model! 👏